beekeeper-path-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/BeekeeperPathSchedulerApiary.java

```diff
@@ -17,6 +17,7 @@
 
 import java.util.TimeZone;
 
+import com.expediagroup.beekeeper.scheduler.apiary.app.PathSchedulerApiaryRunner;
```

Comment:
This import needs to be back to where it was, we put our imports after third party imports.